src/main/java/org/isf/examination/model/PatientExamination.java

```diff
@@ -54,7 +54,7 @@
 @AttributeOverride(name = "lastModifiedBy", column = @Column(name = "PEX_LAST_MODIFIED_BY"))
 @AttributeOverride(name = "active", column = @Column(name = "PEX_ACTIVE"))
 @AttributeOverride(name = "lastModifiedDate", column = @Column(name = "PEX_LAST_MODIFIED_DATE"))
-public class PatientExamination implements Serializable, Comparable<PatientExamination> {
+public class PatientExamination extends Auditable<String> implements Comparable<PatientExamination> {
```

Comment:
Why `Serializable` has been removed too?

Reply:
Three reasons the class doesn't implement `Serializable`:
1. no other class that extends `Auditable` implements `Serializable`
2. the class contains the object `patient` which is a non-serializable field
3. the methods `readObject()` and/or `write Object()` are not defined